lib/tapioca/dsl/compilers/job_iteration.rb

```diff
@@ -39,6 +52,9 @@ def decorate
               expanded_parameters = parameters
             end
 
+            # Sorbet expects optional keyword arguments to be after required keyword arguments.
+            expanded_parameters.sort_by! { |typed_param| PARAM_TYPES_IN_ORDER.index(typed_param.param.class) }
+
```

Comment:
This is fine for now, but I wonder if `rbi` gem should be responsible for generating a method signature to match this ordering, in the future.

@Morriar WDYT?